DOCS-2113 and DOCS-2114 (#11138)
DataDog/documentation: content/en/synthetics/private_locations/_index.md

### Docker

> Private locations are Docker containers that you can install anywhere inside your private network. You can access the [private location worker image][3] on Docker Hub. It can run on a Linux based OS or Windows OS if the [Docker engine][4] is available on your host and can run in Linux containers mode.

Reviewer: Did you mean to change "Docker" to "worker" here?

Author: Yes! The original link sends users to the worker image on Docker Hub so I added "worker" into the CTA.

{{< /site-region >}}

### Permissions

> By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Users with the Datadog Admin and Datadog Standard roles can view private locations, search for private locations, and assign Synthetic tests to private locations. 

Reviewer 2: ```suggestion
By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Users with the Datadog Admin and Datadog Standard roles can view private locations, search for private locations, and assign Synthetic tests to private locations. You can get access to the **Private Locations** page by having your user upgraded to one of these two [default roles][link]. 
```

Reviewer 2: Oh and we should also add the right links on:
- admin role
- standard role
- default roles

> If you have access to the [custom role feature][18], add users to a custom role that includes `synthetics_private_location_read` and `synthetics_private_location_write` permissions for global variables. 

Reviewer 2: ```suggestion
If you have access to the [custom role feature][18], add users to a custom role that includes `synthetics_private_location_read` and `synthetics_private_location_write` permissions for private locations. 
```

{{< /site-region >}}

### Permissions

> By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Admin and Standard users can view private locations, search for private locations, and assign Synthetic tests to private locations. 

Reviewer 2: ```suggestion
By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Users with the Datadog Admin and Datadog Standard roles can view private locations, search for private locations, and assign Synthetic tests to private locations. 
```

> By default, only users with the preset Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Admin and Standard users can view private locations, search for private locations, and assign Synthetic tests to private locations. 

Reviewer: ```suggestion
By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Admin and Standard users can view private locations, search for private locations, and assign Synthetic tests to private locations. 
```

> For more information, see the [Private locations permissions][18] documentation.

Reviewer 2: ```suggestion
If you have access to the custom role feature (link to https://docs.datadoghq.com/account_management/rbac/?tab=datadogapplication#custom-roles), have your user added to a custom role that includes permissions for global variables (`synthetics_private_location_read` and `synthetics_private_location_write` permissions). 
```

## Set up your private location

### Create your private location

In **Synthetic Monitoring** > **Settings** > **Private Locations**, click **Add Private Location**.

{{< img src="synthetics/private_locations/add_pl.png" alt="create a private locations"  style="width:100%;">}}

> **Note**: By default, only Admin users can create private locations. 

Reviewer 2: Let's get rid of this since everything is now mentioned in the permissions section!

Author: ```suggestion
```

> Enable read and write permissions with `synthetics_private_location_read` and `synthetics_private_location_write`. Creating and modifying custom roles is an opt-in Enterprise feature. To get this feature enabled on your account, contact [Datadog support][17].

Reviewer 2: Same comment as above, let's get rid of this!

Author: ```suggestion
```

[14]: /synthetics/browser_tests/?tab=requestoptions
[15]: /synthetics/private_locations/configuration#advanced-configuration
[16]: /agent/
[17]: /synthetics/settings/#private-locations

> [18]: /account_management/rbac/?tab=datadogapplication#custom-roles

Author: ```suggestion
[18]: /account_management/rbac/?tab=datadogapplication#custom-roles
[19]: /account_management/rbac/permissions
```

[14]: /synthetics/browser_tests/?tab=requestoptions
[15]: /synthetics/private_locations/configuration#advanced-configuration
[16]: /agent/

> [17]: /help/

Author: ```suggestion
[17]: /synthetics/settings/#private-locations
[18]: /account_management/rbac/?tab=datadogapplication#custom-roles
```

> [18]: /synthetics/settings/#private-locations

Author: ```suggestion
[18]: /synthetics/settings/#private-locations
[19]: /account_management/rbac/?tab=datadogapplication#custom-roles
```

> [17]: /help/
> [18]: /synthetics/settings/#private-locations

Author: ```suggestion
[17]: /synthetics/settings/#private-locations
[18]: /account_management/rbac/?tab=datadogapplication#custom-roles
```
